pkgs/tools/filesystems/xtreemfs/default.nix

```diff
@@ -9,38 +9,57 @@
 , cmake
 , file
 , fuse
-, jdk
+, jdk8
 , openssl
 , python3
 , valgrind
 , which
 }:
 
 stdenv.mkDerivation {
+  pname = "XtreemFS";
+  # using unstable release because stable (v1.5.1) has broken repl java plugin
+  version = "unstable-2015-06-17";
```

Comment:
Do you think it would a good idea to bump to newer commit? https://github.com/xtreemfs/xtreemfs/commits/master
Than you can skip some of the patches.

Reply:
Not really: newer commits drop Ant in favor of Maven, and I have no idea how to build without network access...

Reply:
We have a couple of examples in the repository, it is not nice but effective...